controls/cis_al2023.yml

```diff
@@ -0,0 +1,2166 @@
+    status: automated
+    rules:
+      - banner_etc_issue
+      - login_banner_text=cis_banners
+
+  - id: 1.7.3
+    title: Ensure remote login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - banner_etc_issue_net
+      - remote_login_banner_text=cis_banners
+
+  - id: 1.7.4
+    title: Ensure permissions on /etc/motd are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_motd
+      - file_owner_etc_motd
+      - file_permissions_etc_motd
+
+  - id: 1.7.5
+    title: Ensure permissions on /etc/issue are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_issue
+      - file_owner_etc_issue
+      - file_permissions_etc_issue
+
+  - id: 1.7.6
+    title: Ensure permissions on /etc/issue.net are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_issue_net
+      - file_owner_etc_issue_net
+      - file_permissions_etc_issue_net
+
+  - id: 1.8
+    title: Ensure updates, patches, and additional security software are installed (Manual)
+    levels:
+      - l1_server
+    status: manual
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: "1.9"
+    title: Ensure system-wide crypto policy is not legacy (Automated)
+    levels:
+      - l1_server
+    status: automated
+    notes: The selected crypto-policy cannot be legacy
+    rules:
+      - configure_crypto_policy
+      - var_system_crypto_policy=default_policy
+
+  - id: 2.1.1
+    title: Ensure time synchronization is in use (Automated)
+    levels:
+      - l1_server
+    status: automated
+    related_rules:
+      - package_chrony_installed
+
+  - id: 2.1.2
+    title: Ensure chrony is configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - chronyd_specify_remote_server
+      - chronyd_run_as_chrony_user
+      - var_multiple_time_servers=amazon
+
+  - id: 2.2.1
+    title: Ensure xorg-x11-server-common is not installed (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - package_xorg-x11-server-common_removed
+
+  - id: 2.2.2
+    title: Ensure avahi is not installed (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - package_avahi_removed
+    related_rules:
+      - service_avahi-daemon_disabled
+
+  - id: 2.2.3
+    title: Ensure a print server is not installed (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - package_cups_removed
+    related_rules:
+      - service_cups_disabled
+
+  - id: 2.2.4
+    title: Ensure DHCP Server is not installed (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - package_dhcp_removed
+
+  - id: 2.2.5
+    title: Ensure DNS Server is not installed (Automated)
```

Comment:
Replace with "title: Ensure a dns server is not installed (Automated)".

Reply:
thanks